game-engine/state-files.md

```diff
@@ -25,20 +26,25 @@
       * range → *The distance to which this can be thrown*
       * count → *The amount of Banana Bombs in this worm's inventory*
       * damageRadius → *The radius to which the impact will destroy dirt blocks and damage worms*
+    * snowballs → *The Snowballs that this worm has at it's disposal. Only applicable to the Technologist worm*
+      * freezeDuration → *The amount of rounds that target worms will be frozen for*
+      * range → *The distance to which this can be thrown*
+      * count → *The amount of Snowballs in this worm's inventory*
+      * freezeRadius → *The radius to which targets will be frozen*
       
       
 * opponents → *The player details of your opponents. An example of an opponent looks like the following:*
   * id → *The Id number of this player*
   * score → *This player's score points currently acquired*
   * currentWormId  → *The Id number of the opponent player's worm that is currently in play*
-  * previousCommand → *The command which this opponent bot supplied to the engine on the last round as a string.  e.g. 'move 20 9'.  Note that it doesn't contain 'C;\<number\>;'*
+  * previousCommand → *The command which this player supplied to the engine on the last round as a string.  e.g. 'move 20 9'.  Note that it doesn't contain 'C;\<number\>;'*
```

Comment:
shouldn't this refer to the opponent?

Reply:
It does, but to the player object here. The wording opponent does make it more clear though 🤔